source/features/last-notification-page-button.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import features from '../feature-manager';
+import looseParseInt from '../helpers/loose-parse-int';
+
+function init(): void | false {
+	const nextButton = select('a[data-hotkey="ArrowRight"]');
+	if (!nextButton) {
+		return false;
+	}
+
+	const lastNotificationPageNode = select('.js-notifications-list-paginator-counts')!.lastChild!;
```

Comment:
Let's use a [content assertion](https://github.com/refined-github/refined-github/pull/6076) here, I think:

```suggestion
	const lastNotificationPageNode = select('.js-notifications-list-paginator-counts')!.lastChild!;
	assertNodeContent(lastNotificationPageNode, /^of \d+$/)
```